From 2abdae7f7fabbfbf8b9087a740d429b0e7ee8d56 Mon Sep 17 00:00:00 2001
From: Biswapriyo Nath <nathbappai@gmail.com>
Date: Wed, 1 Jan 2025 00:00:00 +0000
Subject: [PATCH] native: Include missing standard headers

---
 include/android/configuration.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/include/android/configuration.h b/include/android/configuration.h
index 46c7dfe..a291db0 100644
--- a/include/android/configuration.h
+++ b/include/android/configuration.h
@@ -26,6 +26,7 @@
 #ifndef ANDROID_CONFIGURATION_H
 #define ANDROID_CONFIGURATION_H
 
+#include <stdint.h>
 #include <sys/cdefs.h>
 
 #include <android/asset_manager.h>
-- 
2.48.1

